powerauth-webflow-authentication-approval-sca/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/method/approvalsca/controller/ApprovalScaController.java

```diff
@@ -200,15 +161,14 @@ public ApprovalScaInitResponse initScaApproval(@RequestBody ApprovalScaInitReque
             FormData formData = new FormDataConverter().fromOperationFormData(operation.getFormData());
             ApplicationContext applicationContext = operation.getApplicationContext();
             OperationContext operationContext = new OperationContext(operation.getOperationId(), operation.getOperationName(), operation.getOperationData(), operation.getExternalTransactionId(), formData, applicationContext);
-            ObjectResponse<InitAuthMethodResponse> objectResponse = dataAdapterClient.initAuthMethod(operation.getUserId(), operation.getOrganizationId(), AuthMethod.LOGIN_SCA, operationContext);
+            ObjectResponse<InitAuthMethodResponse> objectResponse = dataAdapterClient.initAuthMethod(operation.getUserId(), operation.getOrganizationId(), AuthMethod.APPROVAL_SCA, operationContext);
             InitAuthMethodResponse initResponse = objectResponse.getResponseObject();
-            // In case client TLS certificate was used during SCA login, use the client TLS certificate for authentication during payment
-            if (initResponse.getCertificateAuthenticationMode() == CertificateAuthenticationMode.ENABLED
-                    && isCertificateUsedForAuthentication(operation.getOperationId())) {
-                String certificateVerificationUrl = initResponse.getCertificateVerificationUrl();
-                logger.debug("Step init succeeded with client certificate, operation ID: {}, authentication method: {}", operation.getOperationId(), getAuthMethodName().toString());
-                return new ApprovalScaInitResponse(true, certificateVerificationUrl);
-            }
+
+            final boolean approvalByCertificateEnabled = initResponse.getCertificateAuthenticationMode() == CertificateAuthenticationMode.ENABLED;
+            setApprovalByCertificateEnabled(approvalByCertificateEnabled);
+
+            logger.debug("Step init succeeded, operation ID: {}, authentication method: {}", operation.getOperationId(), getAuthMethodName().toString());
```

Comment:
I believe that we may omit `.toString()` to improve perf.
```suggestion
            logger.debug("Step init succeeded, operation ID: {}, authentication method: {}", operation.getOperationId(), getAuthMethodName());
```

Reply:
Sure, why not.